src/fairchem/core/trainers/base_trainer.py

```diff
@@ -298,49 +298,49 @@ def load_datasets(self) -> None:
                 self.train_sampler,
             )
 
```

Comment:
basically, this logic to create the validation dataset loader was indented one tab too many to the right, which meant that you had to always have the train dataset loader before this code was run. (expand the code above to see what I'm talking about)